crates/ide-db/src/imports/import_assets.rs

```diff
@@ -536,6 +537,31 @@ fn trait_applicable_items(
         required_assoc_items.insert(assoc);
         Some(assoc_item_trait.into())
     })
+    .filter(|&candidate_trait_id| {
```

Comment:
Oh actually, one more thing, I think we can get the same traits multiple times as this point? Especially for an empty fuzzy string search we'll end up with a trait N times (N being its number of assoc items). So maybe we should dedup / collect into a set once first, then filter out by this?

Reply:
done!